Add blog post for Camel Quarkus 1.1.0 (#477)
apache/camel-website: content/blog/2020/09/camel-quarkus-release-1.1.0/index.md

---
title: "Camel Quarkus 1.1.0 Released"
date: 2020-09-14
authors: ["jamesnetherton"]
categories: ["Releases", "Camel Quarkus"]
preview: "Camel Quarkus 1.1.0 Released"
---

Apache Camel Quarkus 1.1.0 has been released!

We continue to integrate with the latest Camel and [Quarkus](https://quarkus.io/) releases, whilst adding new features and fixing bugs.

For a full overview of the changes see the [1.1.0 milestone details](https://github.com/apache/camel-quarkus/milestone/6?closed=1).

Here are some of the highlights.

## Major component upgrades

* [Camel 3.5.0](https://camel.apache.org/blog/2020/09/Camel35-Whatsnew/)
* Quarkus 1.8.0

## New extensions

Camel Quarkus now has extensions for almost all Camel components. Many new ones have been added in this release, initially with support for JVM mode only. 

In addition, native support has been added to the following extensions:

* [ArangoDB](https://camel.apache.org/camel-quarkus/latest/reference/extensions/arangodb.html)
* [AS2](https://camel.apache.org/camel-quarkus/latest/reference/extensions/as2.html)

> * [AWS2 Lambda](https://camel.apache.org/camel-quarkus/latest/reference/extensions/aws2-lambda.html)

Reviewer: I believe there is also AWS2-STS

Author: Thanks! Added now.

---
title: "Camel Quarkus 1.1.0 Released"
date: 2020-09-14
authors: ["jamesnetherton"]
categories: ["Releases", "Camel Quarkus"]
preview: "Camel Quarkus 1.1.0 Released"
---

Apache Camel Quarkus 1.1.0 has been released!

We continue to integrate with the latest Camel and [Quarkus](https://quarkus.io/) releases, whilst adding new features and fixing bugs.

For a full overview of the changes see the [1.1.0 milestone details](https://github.com/apache/camel-quarkus/milestone/6?closed=1).

Here are some of the highlights.

## Major component upgrades

* [Camel 3.5.0](https://camel.apache.org/blog/2020/09/Camel35-Whatsnew/)
* Quarkus 1.8.0

## New extensions

> Camel Quarkus now has extensions for almost all Camel components. Many new ones have been added in this release, initially with support for JVM mode only. 

Reviewer 2: ```suggestion
We have added a massive amount of 174 new extensions in this release. This is because we hurry up to become [the only and default runtime](http://camel.465427.n5.nabble.com/camel-k-switch-to-Quarkus-as-default-framework-for-integrations-td5869959.html) of our sibling [Camel K](https://camel.apache.org/camel-k/latest/). 

159 of the new extensions have initially support for JVM mode only.
```

> In addition, native support has been added to the following extensions:

Reviewer 2: ```suggestion
In addition, native support has been added to the following 14 extensions:
```

* [Vert.x WebSocket](https://camel.apache.org/camel-quarkus/latest/reference/extensions/vertx-websocket.html)
* [Weather](https://camel.apache.org/camel-quarkus/latest/reference/extensions/weather.html)

You can browse the full list of supported extensions over at the [extensions reference](https://camel.apache.org/camel-quarkus/latest/reference/index.html).

## Less use of reflection

Over the past months, we have gradually reduced the number of Camel classes that were needlessly required to be registered for reflection. Thanks to Camel 3.5.0, 
we have been able to eliminate all of these remaining cases and take adavantage of the resulting performance boost.

## SNAPSHOT builds

The project has started publishing SNAPSHOT releases from the `master` branch (builds with stable Camel & Quarkus releases), the `camel-master` branch (builds with the latest Camel SNAPSHOT) and also from the `quarkus-master` branch (builds with the latest Quarkus SNAPSHOT). There are more details about this here:

https://camel.apache.org/camel-quarkus/latest/contributor-guide/ci.html#_snapshot_deploy_build

> 

Reviewer 2: ```suggestion

## We ❤️ new contributors!

We were happy to welcome as many as four new contributors in this release period. [Lukáš](https://github.com/llowinge) improved some of our tests, [Zineb](https://twitter.com/ZinebBendhiba) and [Marcel](https://twitter.com/JeansenML) were brave enough to deliver two native extensions each and [Pooja](https://github.com/PoojaChandak) invested her time in improving our documentation.
Many thanks for the contributions!
```
